packages/tabsheet/src/vaadin-tabsheet-mixin.d.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright (c) 2019 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import type { Constructor } from '@open-wc/dedupe-mixin';
+import type { DelegateStateMixinClass } from '@vaadin/component-base/src/delegate-state-mixin.js';
+import type { Tab } from '@vaadin/tabs/src/vaadin-tab.js';
```

Comment:
This makes the mixin depend on Polymer-based tabs. Should we change `items` type to `HTMLElement[]`?

Reply:
Done. Made `Tab` in the mixin a generic type

Reply:
Looks good, I will do the same for `AccordionPanel` in my PR.